### PR TITLE
(maint) correct pathways for @puppet scoping

### PR DESCRIPTION
**Pull request by KNBAnderson**

### CODE REVIEW

**Comment by jonathannewman** [comment]
Looks like this needs a rebase.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@puppet/ivy-tabs",
-  "version": "4.0.1",
+  "version": "4.0.2",
```
Comment (jonathannewman) [inline package.json:3]: Is this version automatically updated when npm publishes?

Comment (KNBAnderson) [reply package.json:3]: No I did this manually

**Comment by jonathannewman** [comment]
LGTM! Nice

**Review by jonathannewman — APPROVED** [review]

**Pull request merged**